rest_framework/authtoken/admin.py

```diff
@@ -45,6 +44,13 @@ def get_object(self, request, object_id, from_field=None):
         except (queryset.model.DoesNotExist, User.DoesNotExist, ValidationError, ValueError):
             return None
 
+    def get_search_fields(self, request):
+        if hasattr(self, 'search_fields'):
+            return self.search_fields
+        user_model = Token._meta.get_field('user').related_model
```

Comment:
Can we use the `get_user_model()` helper from Django instead?

Reply:
Good point. It was even used and assigned to `User` above, but I somehow missed it.

Reply:
Codebase seems to be using mostly old-style string interpolation, so I stuck with that as @p-r-a-v-i-n also suggested.

Marking thread resolved.